### ISSUE TITLE
Virus Detections (Windows Defender)

### ISSUE THREAD

**Issue by martin951** [richrobber2/canvas-zoom#65]
1: Trojan:Win32/Phonzy.B!ml > index-4ad95a66.js
2: Trojan:Script/Wacatac.B!ml > dockerfile-d67bbd50.js

**Comment by daswer123** [comment]
This is the second time people have written to me about this, and it's very strange. These files, as well as the templates folder, are the modified frontend part of gradio, which is needed for the extension to work. I compiled from the official gradio repo, and everything I change can be seen in the dist/js folder, just in case I uploaded these files to virustotal.

index-4ad95a66.js
https://www.virustotal.com/gui/file/8ce417d3c5c825034be2190a64ad9d4617900c42731437f797e83499b3d5d047?nocache=1

dockerfile-d67bbd50.js
https://www.virustotal.com/gui/file/75762a2a0a33bc7ddfcf5b0caf89cb681a380d424c4c449dbaf559699cc33b54?nocache=1
